airflow-core/src/airflow/ui/src/pages/XCom/XCom.tsx

```diff
@@ -34,6 +36,49 @@ const columns: Array<ColumnDef<XComResponse>> = [
     enableSorting: false,
     header: "Key",
   },
+  {
+    accessorKey: "dag_id",
```

Comment:
Can dag_id, run_id and task_id be hidden similar to how they are not rendered in events page?

https://github.com/apache/airflow/blob/6a703dd1f795784e446742dddf412b0fe3b4e4a0/airflow-core/src/airflow/ui/src/pages/Events/Events.tsx#L44

Reply:
dag_id, run_id, and task_id are always defined. 

```
class XComResponse(BaseModel):
    """Serializer for a xcom item."""

    key: str
    timestamp: datetime
    logical_date: datetime | None
    map_index: int
    task_id: str
    dag_id: str
    run_id: str

```

Reply:
Sorry, I meant on the basis of them defined in the URL like how it's in the linked events page implementation. 

```js
const { dagId, runId, taskId } = useParams();
```